server.js

```diff
@@ -59,6 +59,12 @@ const server = async () => {
     transports: [new transports.Http(httpTransportOptions)]
   })
 
+  cloudinary.config({
+    cloud_name: 'scheduleapp',
+    api_key: process.env.CLOUDINARY_API_KEY,
+    api_secret: process.env.CLOUDINARY_API_SECRET
+  })
```

Comment:
use config object here, in process.env we'll only have NODE_ENV, PORT, DB_URI and SECRET_KEY, nothing more.

Reply:
use config object, the process.env will only have PORT, NODE_ENV, DB_URI and SECRET_KEY. All other config data will be stored on the config object. Also, make sure that CLOUDINARY_API_KEY and CLOUDINARY_API_SECRET are present in the db